### ISSUE TITLE
Treeview in AOR: Accounts item is always at the top

### ISSUE THREAD

**Issue by likhobory** [salesagility/SuiteCRM#2830]
'Accounts' item is always at the top with **ANY** lang. pack. Tested with English, Russian and Portuguese languages. This branch has been used: https://github.com/shogunpol/SuiteCRM/tree/bug636

It's a bug or a feature?

![tview9](https://cloud.githubusercontent.com/assets/5983672/21471708/d5867ad2-cacb-11e6-9984-4f70f67c4778.png)

See also #636

**Comment by horus68** [comment]
This should be considered a bub.
Also note that in portuguese "Contas" (accounts) should be on top of "Contas : membros" (Accounts : members) 
![contas](https://cloud.githubusercontent.com/assets/3055443/21483205/682d3bf0-cb76-11e6-99da-73931fa6af67.png)

**Comment by shogunpol** [comment]
@likhobory , @horus68 , are you suggest to do kind of alphabetical sorting? In issue #636 what i tried to do it was to name all used labels, it was around 150 of them, i did not tried nothing else. So please tell me more about this suggestion, so we can process and decide to what to do.

**Comment by horus68** [comment]
@shogunpol the alphabetical sorting would be a useful feature so its easier to find a module.
Now it looks like "Accounts" is not ordered but the others are (including Account : Members)

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Treeview in AOR: Accounts item is always at the top](https://trello.com/c/JBtHYOWw/44-treeview-in-aor-accounts-item-is-always-at-the-top)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
